google/cloud/ndb/_cache.py

```diff
@@ -306,6 +306,32 @@ def __init__(self, options):
         self.todo = {}
         self.futures = {}
 
+    def done_callback(self, cache_call):
+        """Process results of call to global cache.
```

Comment:
Per [PEP 257](https://www.python.org/dev/peps/pep-0257/#id17), there should be a blank line between the summary and the body of a multi-line docstring.

Reply:
38e3936